chpl/chpl-api/e2e/data/2014chplids-test-data.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "certifiedProductId":4770,
```

Comment:
I did not find where this certifiedProductId is used in the tests

Reply:
This is the data using for /{certifiedProductId} and /{certifiedProductId}/measures end points